10-validation.md

```diff
@@ -1,79 +1,77 @@
 
-1. In the Azure Portal, navigate to your AKS cluster resource.
-1. Click _Logs_ to see and query log data.
-   :bulb: There are several examples on the _Kubernetes Services_ category.
 
-## Validate Azure Monitor for containers (Prometheus Metrics)
+> :warning: Depending on your actual location traffic might flow different for you. But having two incidents ensure you test at least one failover.
 
-Azure Monitor is configured to [scrape Prometheus metrics](https://docs.microsoft.com/azure/azure-monitor/insights/container-insights-prometheus-integration) in your cluster. This reference implementation is configured to collect Prometheus metrics from two namespaces, as configured in [`container-azm-ms-agentconfig.yaml`](./cluster-baseline-settings/container-azm-ms-agentconfig.yaml). There are two pods configured to emit Prometheus metrics:
+![Azure Monitor Dashboard that helps to observe the `East US 2` region outage simulation and how the traffic flowed from `East US 2` to `Central US`](images/azure-monitor-dashboard-1st-failover.png)
 
-- [Treafik](./workload/traefik.yaml) (in the `a0008` namespace)
-- [Kured](./cluster-baseline-settings/kured-1.4.0-dockerhub.yaml) (in the `cluster-baseline-settings` namespace)
+> Note: from the Inbound Multi Cluster Traffic Flow Count and Azure AppFw Health Dashboaard Metrics: :large_blue_circle: East US 2 :red_circle: Central US
 
-### Steps
+### Second incident 4:56PM UTC time, `East US 2` resumed its operations but `Central US` is about to go down as well
 
-1. In the Azure Portal, navigate to your AKS cluster resource group (`rg-bu0001a0042-03` or `rg-bu0001a0042-04`).
-1. Select your Log Analytic Workspace resource.
-1. Click _Saved Searches_.
+Now `East US 2` region is back after the outage. Every 30 seconds, Azure Front Door sample the rountrip latencies against its backend pools using the configured health probe, and once again determines `East US 2` is the best candidate as this normalized its operations. Traffic starts flowing at 4:56PM the other way around from `Central US` to `East US 2`, and now everything is back to normal. At the same time, you can observe an new outage, same symptom as before but it is now in `Central US` where its compute units are close to `0` and healthyness is about to drop to `0%` a few seconds after. It does not represent a threat since just a moment ago traffic had already flowed to `East US 2`. Around 5:15PM all regions are operative, and the clients never suffered the consequences of these multiple total region outages.
```

Comment:
```suggestion
Now `East US 2` region is back after the outage. Every 30 seconds, Azure Front Door samples the roundtrip latencies against its backend pools using the configured health probe, and once again determines `East US 2` is the best candidate as this normalized its operations. Traffic starts flowing at 4:56PM the other way around from `Central US` to `East US 2`, and now everything is back to normal. At the same time, you can observe a new outage, same symptom as before, but it is now in `Central US` where its compute units are close to `0` and healthiness is about to drop to `0%` a few seconds after. It does not represent a threat since just a moment ago traffic had already flowed to `East US 2`. Around 5:15PM all regions are operative, and the clients never suffered the consequences of these multiple total region outages.
```